index.d.ts

```diff
@@ -0,0 +1,39 @@
+// copied from Redux so that we don't have to add it as a dependency
+interface Action<T = any> {
```

Comment:
We don't have `Redux` as a dependency and I wasn't sure if we should add it just for this. We probably should.. Thoughts @bdwain ?

Reply:
can  you add it as a peer  dependency?  and a dev dep.

Reply:
Sure can.

Reply:
Ah, yarn only wants it in one, not both. It refuses to add the second one. I'll add it just as a peer dep.

Reply:
Well.. Nevermind, it works if I add it as a peer FIRST and then as a dev. That's weird.